app/jobs/notify_followers_of_new_post_job.rb

```diff
@@ -61,4 +63,12 @@ def view_post(post_id)
     url = Rails.application.routes.url_helpers.post_url(post_id, host: host, protocol: 'https')
     "<a href='#{url}'>View it here</a>."
   end
+
+  def blocked_user_ids(post)
+    blocked = Block.where(blocked_user_id: post.author_ids).where("hide_them >= ?", Block::POSTS)
+    blocked = blocked.select(:blocking_user_id).distinct.pluck(:blocking_user_id)
+    blocking = Block.where(blocking_user_id: post.author_ids).where("hide_me >= ?", Block::POSTS)
+    blocking = blocking.select(:blocked_user_id).distinct.pluck(:blocked_user_id)
+    (blocked + blocking).uniq
```

Comment:
Would it be better to check the blocked post caches of favorited users here? At the point where this is going out they probably haven't regenerated yet, which means a bunch of seperate db queries where this just queries twice -- and actually could probably query only once -- but it means we're not skipping the cache.